packages/casting/test/net-access-fixture.js

```diff
@@ -0,0 +1,196 @@
+const { stringify: jq } = JSON;
+
+/**
+ * @file to regenerate
+ *   1. set RECORDING=true in test-interpose-net-access.js
+ *   2. run: yarn test test/test-test-interpose-net-access.js --update-snapshots
+ *   3. for each map in test-test-interpose-net-access.js.md, copy it and
+ *   4. replace all occurences of => with : and paste as args to Object.fromEntries()
+ *   5. change RECORDING back to false
+ */
+export const web1 = new Map([
+  [
+    jq([
+      'https://emerynet.rpc.agoric.net/',
+      {
+        method: 'POST',
+        body: jq({
+          id: 1208387614,
+          method: 'no-such-method',
+          params: [],
+          jsonrpc: '2.0',
+        }),
+        headers: { 'Content-Type': 'application/json' },
+      },
+    ]),
+    {
+      error: {
+        code: -32601,
+        message: 'Method not found',
+      },
+      id: 1,
```

Comment:
This `id` should match the one on line 18 it is replying to, in order to conform to the [JSON-RPC response object spec](https://www.jsonrpc.org/specification#response_object).